### PR TITLE
home-assistant 2021.5 dependency bumps

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

https://github.com/home-assistant/core/releases/tag/2021.5.0b5

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mweinelt** [comment]
@graham33 I added you as maintainer for authcaptureproxy, as you packaged that in #113855. Alas there is a newer version out there, the packaging changed a bit, and it was indeed required for this teslajsonpy update.

**Review by mvnetbiz — APPROVED** [review]
Bellows works good, I can't test anything else but the other changes look typical. :+1:

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 121457` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>11 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.authcaptureproxy</li>
    <li>python38Packages.bellows</li>
    <li>python38Packages.pysonos</li>
    <li>python38Packages.simplisafe-python</li>
    <li>python38Packages.teslajsonpy</li>
    <li>python39Packages.authcaptureproxy</li>
    <li>python39Packages.bellows</li>
    <li>python39Packages.pysonos</li>
    <li>python39Packages.simplisafe-python</li>
    <li>python39Packages.teslajsonpy</li>
  </ul>
</details>

File: pkgs/development/python-modules/authcaptureproxy/default.nix [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    yarl
+  ];
+
+  checkInputs = [
+    pytest-asyncio
+    pytestCheckHook
+  ];
+
+  meta = with lib; {
+    description = "A Python project to create a proxy to capture authentication information from a webpage";
```
Comment (lukegb) [inline pkgs/development/python-modules/authcaptureproxy/default.nix:55]:
```suggestion
    description = "A proxy to capture authentication information from a webpage";
```

Comment (mweinelt) [reply pkgs/development/python-modules/authcaptureproxy/default.nix]: Applied, thanks!

**Review by graham33 — APPROVED** [review]
These changes look good to me from a brief review.

I was able to test teslajsonpy and authcaptureproxy to some extent and they seem fine.

**Pull request merged**